.circleci/config.yml

```diff
@@ -25,8 +24,9 @@ executors:
 commands:
   install-deps:
     steps:
-      - go/install-ssh
-      - go/install: {package: git}
+      - run: |
+          sudo apt update
+          sudo apt install python-is-python3
```

Comment:
Because we use `python` in our scripts, but ubuntu has `python2` and `python3`. I would fix our scripts, but macos just uses `python` and I'm apparently not allowed to break builds people with macs.